pallets/liquid-staking/src/lib.rs

```diff
@@ -527,10 +510,7 @@ pub mod pallet {
             #[pallet::compact] amount: BalanceOf<T>,
         ) -> DispatchResultWithPostInfo {
             T::RelayOrigin::ensure_origin(origin)?;
-            InsurancePool::<T>::try_mutate(|v| -> DispatchResult {
-                *v = v.checked_sub(amount).ok_or(ArithmeticError::Underflow)?;
-                Ok(())
-            })?;
+            T::XCM::reduce_insurance_pool(amount)?;
```

Comment:
insurance pool related stuff should still be put inside liquid staking as above

Reply:
replied above